tutorials/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -38,8 +38,11 @@ Here is a complete class example based on these guidelines:
     signal state_changed(previous, new)
 
     @export var initial_state: Node
+
+    static var max_instance_count = 10
     var is_active = true:
         set = set_is_active
+    var _current_state: Node
```

Comment:
It would be nice if the private variable was used in the class, otherwise a warning would be produced.